tests/suites/test_suite_ssl.function

```diff
@@ -1186,6 +1186,44 @@ int psk_dummy_callback( void *p_info, mbedtls_ssl_context *ssl,
 #define SSL_CID_LEN_MIN MBEDTLS_SSL_CID_OUT_LEN_MAX
 #endif
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+static int psa_cipher_encrypt_helper( mbedtls_ssl_transform *transform,
```

Comment:
Suggestion: I think it would be better to always declare this function, and then have the `#if defined(MBEDTLS_USE_PSA_CRYPTO)` in the body, with the trivial implementation based on `mbedtls_cipher_crypt` in the `#else` branch. That way, this function can be called without worrying about which case we're in (and without duplicating the list of arguments, which is error-prone).